dev/doc/changes.md

```diff
@@ -35,6 +35,18 @@
 - A few functions change their interfaces to take benefit of a new
   abstraction level `LStream` for streams with location function.
 
+- Grammar extensions now require to specify whether they create a level or they
+  reuse an existing one. In addition to the Gramlib API changes, GRAMMAR EXTEND
+  stanzas may need a few tweaks. Their grammar was changed so that level and
```

Comment:
```suggestion
- Grammar extensions now require specifying whether they create a level or they
  reuse an existing one. In addition to the Gramlib API changes, GRAMMAR EXTEND
  constructs may need a few tweaks. Their grammar was changed so that level and
```

"stanza" is poetry term, AFAIK it is not generally used as a computer science term.  It doesn't appear elsewhere in this document, either.

Reply:
https://en.wikipedia.org/wiki/Stanza_(computing)

Reply:
I confirm it is in common use in a few fields I interact with [build systems, HCI, etc...]

Reply:
The same term is actually used a bit further down in the context in ARGUMENT EXTEND.

Reply:
I stand by my comment. This not a _generally_ used computer science term.  The wikipedia page is all of two sentences first created in 2016.  Each of the three references has their own language that defines _stanza_.  The Dune documentation does not, to its detriment (I recall reading and thinking, "Well, what the heck is a 'stanza'?".

We try to use common terminology and avoid needless jargon to make documentation easy to understand.

> The same term is actually used a bit further down in the context of ARGUMENT EXTEND.

Clearly a failure of the review process!  I wonder who might have submitted such wording?